Makefile

```diff
@@ -54,9 +55,9 @@ export DOCKER_REGISTRY  ?= itisfoundation
 .PHONY: help
 help: ## help on rule's targets
 ifeq ($(IS_WIN),)
-	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+	@awk --posix 'BEGIN {FS = ":.*?## "} /^[[:alpha:][:space:]_-]+:.*?## / {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
```

Comment:
long live awk ;) welcome in the 2000s 🤣

Reply:
Be careful. I got a PhD using sed, awk, vi and cvs.

Reply:
well he was using awk from 1996 until recently...

Reply:
what can i say ... I like the classics :-)